lib/generators/hyrax/templates/db/migrate/20240606205215_create_hyrax_flexible_schemas.rb.erb

```diff
@@ -0,0 +1,9 @@
+class CreateHyraxFlexibleSchemas < ActiveRecord::Migration<%= migration_version %>
```

Comment:
Should this have a `unless Hyrax.config.flexible?` guard?

Reply:
I'm not sure. Does it hurt to have the models present but not use them?